phpBB/styles/prosilver/template/ucp_pm_viewfolder.html

```diff
@@ -1,130 +1,70 @@
-							<a href="{messagerow.U_MCP_REPORT}">
-								<i class="icon fa-exclamation fa-fw icon-red" aria-hidden="true"></i><span class="sr-only">{PM_REPORTED}</span>
-							</a>
-						<!-- ENDIF --> <!-- IF messagerow.ATTACH_ICON_IMG --><i class="icon fa-paperclip fa-fw" aria-hidden="true"></i> <!-- ENDIF --><br />
-						<!-- IF S_SHOW_RECIPIENTS -->{L_MESSAGE_TO} {messagerow.RECIPIENTS}<!-- ELSE -->{L_MESSAGE_BY_AUTHOR} {messagerow.MESSAGE_AUTHOR_FULL} &raquo; {messagerow.SENT_TIME}<!-- ENDIF -->
-
-						</div>
-					</dt>
-					<!-- IF S_SHOW_RECIPIENTS --><dd class="info"><span>{L_SENT_AT}{L_COLON} {messagerow.SENT_TIME}</span></dd><!-- ENDIF -->
-					<!-- IF S_UNREAD --><dd class="info"><!-- IF messagerow.FOLDER --><a href="{messagerow.U_FOLDER}">{messagerow.FOLDER}</a><!-- ELSE -->{L_UNKNOWN_FOLDER}<!-- ENDIF --></dd><!-- ENDIF -->
-					<dd class="mark"><input type="checkbox" name="marked_msg_id[]" value="{messagerow.MESSAGE_ID}" /></dd>
-				</dl>
-			</li>
-		<!-- END messagerow -->
-
-		</ul>
-	<!-- ELSE -->
-		<p><strong>
-			<!-- IF S_COMPOSE_PM_VIEW and S_NO_AUTH_SEND_MESSAGE -->
-				<!-- IF S_USER_NEW -->{L_USER_NEW_PERMISSION_DISALLOWED}<!-- ELSE -->{L_NO_AUTH_SEND_MESSAGE}<!-- ENDIF -->
-			<!-- ELSE -->
-				{L_NO_MESSAGES}
-			<!-- ENDIF -->
-		</strong></p>
-	<!-- ENDIF -->
-
-	<!-- IF FOLDER_CUR_MESSAGES neq 0 -->
-		<fieldset class="display-actions">
-			<div class="left-box"><label for="export_option">{L_EXPORT_FOLDER}{L_COLON} <select name="export_option" id="export_option"><option value="CSV">{L_EXPORT_AS_CSV}</option><option value="CSV_EXCEL">{L_EXPORT_AS_CSV_EXCEL}</option><option value="XML">{L_EXPORT_AS_XML}</option></select></label> <input class="button1 button button-form-bold" type="submit" name="submit_export" value="{L_GO}" /><br /></div>
-			<select name="mark_option">{S_MARK_OPTIONS}{S_MOVE_MARKED_OPTIONS}</select> <input class="button1 button button-form-bold" type="submit" name="submit_mark" value="{L_GO}" />
-			<div><a href="#" onclick="marklist('viewfolder', 'marked_msg', true); return false;">{L_MARK_ALL}</a> &bull; <a href="#" onclick="marklist('viewfolder', 'marked_msg', false); return false;">{L_UNMARK_ALL}</a></div>
-		</fieldset>
-
-		<hr />
-
-		<div class="action-bar bottom">
-			<!-- INCLUDE display_options.html -->
-			<input type="hidden" name="cur_folder_id" value="{CUR_FOLDER_ID}" />
-
-			<div class="pagination">
-				{TOTAL_MESSAGES}
-				<!-- IF .pagination -->
-					<!-- INCLUDE pagination.html -->
-				<!-- ELSE -->
-					 &bull; {PAGE_NUMBER}
-				<!-- ENDIF -->
-			</div>
-		</div>
-	<!-- ENDIF -->
-
-		</div>
+</div>
+
+<div class="action-bar bottom">
+	{% include 'display_options.html' %}
+	<input type="hidden" name="cur_folder_id" value="{{ CUR_FOLDER_ID }}" />
+
+	<div class="pagination">
+		{{ TOTAL_MESSAGES }}
+		{% if pagination %}
+			{% include 'pagination.html' %}
+		{% else %}
+			&bull; {{ PAGE_NUMBER }}
+		{% endif %}
 	</div>
+</div>
+{% endif %}
+
+</div>
```

Comment:
There appears to be a missing closing `</div>` tag that needs to go here, so you have `</div></div>`. Viewing the fully rendered source code reveals a closing DIV here gets lost. This is also true for the end of `ucp_pm_viewmessage.html`

I suggest moving these closing `</div></div>` tags into the top of `ucp_pm_message_footer.html` file because these two tags were opened in the message_header file, so they should be closed in the message_footer file, and that will also keep these viewfolder and viewmessage files cleaner, and with no stray closing tags in them.